generators/app/templates/gulp/_building.js

```diff
@@ -21,11 +25,28 @@ gulp.task('build', buildDependencies, function () {
     .pipe($.size({showFiles: true}));
 });
 
-gulp.task('clean', function () {
-  return gulp.src(['.tmp', paths.dist + '/*'])
+gulp.task('clean', ['clean-styles','clean-templates'], function () {
```

Comment:
gulp tasks: please leave the original gulp tasks in tact, I assume that some of the gulp tasks break because of these changes.